HomeworkOnlineStore/HomeworkOnlineStore/Models/Products.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HomeworkOnlineStore.Models
+{
+    internal class Product
+    {
+        private double _price;
+        private string _name;
+        private int _productID;
```

Comment:
Can be converted to  public double Price { get;  set; } or  public double Price { get;  private set; }